src/FsAutoComplete.Core/Commands.fs

```diff
@@ -238,6 +238,33 @@ type Commands (serialize : Serializer, backgroundServiceEnabled) =
         let loader = Dotnet.ProjInfo.Workspace.Loader.Create(config)
         loader, checker.CreateFCSBinder(NETFrameworkInfoProvider.netFWInfo, loader)
 
+    let pageForLint (code: string) =
+        let codeNumber = code.Substring (2) |> int
+        match codeNumber with
+        | 1 | 2 | 3 -> Some "http://fsprojects.github.io/FSharpLint/TupleFormatting.html"
+        | 4 | 5 | 6 | 7 -> Some "http://fsprojects.github.io/FSharpLint/PatternMatchFormatting.html"
+        | 8 | 9 -> Some "http://fsprojects.github.io/FSharpLint/Spacing.html"
+        | 10 (* | 11 (not present?) *) | 12 -> Some "http://fsprojects.github.io/FSharpLint/Formatting.html"
```

Comment:
added https://github.com/fsprojects/FSharpLint/issues/362 to move this in FSharpLint vNext